superduperdb/base/datalayer.py

```diff
@@ -69,15 +69,13 @@ def __init__(
         databackend: BaseDataBackend,
         metadata: MetaDataStore,
         artifact_store: ArtifactStore,
-        distributed_client=None,
+        compute: ComputeBackend = LocalComputeBackend(),
     ):
         """
-        :param databackend: databackend object containing connection to Datastore
-        :param metadata: metadata object containing connection to Metadatastore
-        :param artifact_store: artifact_store object containing connection to
-                               Artifactstore
-        :param distributed_client: distributed_client object containing connection to
-                                   ``dask`` cluster (leave alone)
+        :param databackend: object containing connection to Datastore
```

Comment:
object here is redundant 
in python language everything is object 

so writing object explicitly will be redundant :)

Reply:
This is to differentiate from strings like "URI for .."